husky_description/urdf/dual_arm_husky.urdf.xacro

```diff
@@ -98,9 +98,9 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <!-- Interial link stores the robot's inertial information -->
   <link name="inertial_link">
     <inertial>
-      <mass value="33.455"/>
-      <origin xyz="-0.08748 -0.00085 0.09947"/>
-      <inertia ixx="0.6022" ixy="-0.02364" ixz="-0.1197" iyy="1.7386" iyz="-0.001544" izz="2.0296"/>
+      <mass value="48.39054"/>
+      <origin xyz="-0.0648 -0.00059 0.07895"/>
+      <inertia ixx="2.1634275613800003" ixy="-0.055452352110000006" ixz="-0.41083822026" iyy="3.5314689495400002" iyz="-0.004366145530000001" izz="4.4759585875" />
```

Comment:
That is correct - husky_navigation was not used on the robot before shipping. The robotiq grippers should work in gazebo. They do not move with ros_control however, they move using direct joint state publisher commands. 

Are you seeing the plugins via source or debs? If its debs its most likely the ones distributed from robotiq themselves - if its by source you can check doing "git remote show origin"